lib/decanter/value_parser/datetime_parser.rb

```diff
@@ -5,8 +5,8 @@ class DateTimeParser < Base
       allow DateTime
 
       parser do |name, val, options|
-        parse_format = options.fetch(:parse_format, '%m/%d/%Y')
-        ::Date.strptime(val, parse_format)
+        parse_format = options.fetch(:parse_format, '%m/%d/%Y %I:%M:%S %p')
+        ::DateTime.strptime(val, parse_format)
```

Comment:
correct, but they shouldn't be using datetime parser for that should they ?
:)

On Wed, Apr 13, 2016 at 11:59 AM, Ryan Francis <notifications@github.com>
wrote:

> In lib/decanter/value_parser/datetime_parser.rb
> <https://github.com/LaunchPadLab/decanter/pull/21#discussion_r59585164>:
>
> > @@ -5,8 +5,8 @@ class DateTimeParser < Base
> >        allow DateTime
> >
> >        parser do |name, val, options|
> > -        parse_format = options.fetch(:parse_format, '%m/%d/%Y')
> > -        ::Date.strptime(val, parse_format)
> > +        parse_format = options.fetch(:parse_format, '%m/%d/%Y %I:%M:%S %p')
> > +        ::DateTime.strptime(val, parse_format)
>
> One thing to keep in mind is that any apps that are currently using
> DateTimeParser to parse a string like "05/18/2015" will need to switch to
> use DateParser instead as this would cause an "Invalid Date" error.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/LaunchPadLab/decanter/pull/21/files/2ddcbee969ed8e206c7c824140d18195cc742e43#r59585164>
>



-- 

*Dave Corwin*
LaunchPad Lab <http://launchpadlab.com/> - Developer
*We Build Apps That Grow Your Business*

tele: 773-405-8401  |  twitter: @davidmcorwin